.github/workflows/build.yml

```diff
@@ -45,10 +45,10 @@ jobs:
           */*/node_modules
           packages/boa/pybind11
           packages/boa/pyodide
-        key: ${{ runner.os }}-${{ matrix.node_version }}-${{ hashFiles('**/package-lock.json,packages/boa/pybind11/Makefile,packages/boa/pyodide/Makefile') }}
+        key: ${{ runner.os }}-${{ matrix.node_version }}-yarn-${{ hashFiles('**/yarn.lock', 'packages/boa/pybind11/Makefile', 'packages/boa/pyodide/Makefile') }}
```

Comment:
Why update the key with appended yarn?

Reply:
YARN is a completely different package manager and the cache is not compatible.

Reply:
But the file is different so that the key has been updated, though.